js/test/Helper.js

```diff
@@ -229,30 +229,30 @@ describe('Helper', function () {
             cleanup();
         });
 
+/* TODO test fails in jsDOM version 17 - document.cookie remains empty
```

Comment:
Hmm first of all, does this still apply? You updated to v26 and mention v17 here, so at least the version may need bumping (and ideally a link to a bug report for JsDom?)?

Secondly, I don't like commented code for code style reasons, but if this still applies, you could maybe just change the `describe` of this test ~~to `xdescribe`~~ (ah this was jasmine, apparently mocha [also has ways for [doing it in a better way though](https://stackoverflow.com/questions/32723167/how-to-programmatically-skip-a-test-in-mocha)). https://mochajs.org/#inclusive-tests

Ah and this test is missing an `it`  is not it? IMHO=, the best structure is always to have such a thing, but this is stylistic refactoring that we can do much later anyway.

Reply:
Ah now re-read and saw you wrote:
> document.cookie can be set and read, but remains empty - documentation suggests use of cookies is still supported, but I didn't get it working for our use case

If this is still a thing we need to fiddle out, IMHO, best is to create a new issue and link it in the source code, so we can tackle this later. Just so we do not forget it (Code TODOs are otherwise easily forgotten…).

Reply:
I'm unsure whether it is a regression or we maybe just need to do stuff differently and don't know if, like mocking a cookie jar.
That's why I ask to keep track of this for later. Of course we can merge out like this and later adjust it.